score_sheets.tex

```diff
@@ -161,6 +161,12 @@
 \input{scoresheets/EEGPSR.tex}
 \end{scoresheet}
 
+% Object Configuration
```

Comment:
Wrong place. The test is a Stage I test and is located between EEGPSR and Open Challenge which are Stage II